### ISSUE TITLE
False-positive: method can be dropped as it only calls parent's one

### ISSUE THREAD

**Issue by b1rdex** [kalessil/phpinspectionsea#310]
```php
interface A
{
    public function foo(): bool;
}

class B
{
    public function foo()
    {
        return true;
    }
}

class C extends B implements A
{
    public function foo(): bool
    {
        return parent::foo();
    }
}
```

`C::foo` is marked as can be dropped. No, it can't, because `B::foo` doesn't have `bool` signature and will break interface constraints.

This looks like bad design, but in my case `B` is framework code but `A` and `C` is app code, so I can't make `B implements A`.

**Comment by kalessil** [comment]
Then we have to re-check returned type hints now =)

**Comment by kalessil** [comment]
Fixed, a fresh binary pushed as well.

**Comment by funivan** [comment]
@kalessil I will check)

**Comment by kalessil** [comment]
@funivan : since we have travis, we could probably start building nignhly versions of the plugin.

**Comment by funivan** [comment]
@kalessil it would be great. And provide repository for the nightly builds. If you remember I provided a link to the "Upsource EAP plugin".
